lib/src/signed_video_tlv.c

```diff
@@ -192,12 +192,18 @@ encode_general(signed_video_t *self, uint8_t *data)
   size_t data_size = 0;
   uint32_t gop_counter = gop_info->global_gop_counter + 1;
   uint16_t num_nalus_in_gop_hash = gop_info->num_nalus_in_gop_hash;
-  const uint8_t version = 1;
+  uint8_t version = 2;
+  int64_t timestamp = self->gop_info->timestamp;
+  uint8_t flags = 0;
 
   // Get size of data
   data_size += sizeof(version);
   data_size += sizeof(gop_counter);
   data_size += sizeof(num_nalus_in_gop_hash);
+  data_size += sizeof(flags);
+  if (gop_info->has_timestamp) {
+    data_size += sizeof(timestamp);
+  }
   data_size += SV_VERSION_BYTES;
```

Comment:
Put these in the same order as they are written for easy reading. Hence, put the new code after `SV_VERSION_BYTES`